file-formats/doma/examples/z_aff_example_doma.doma.json

```diff
@@ -0,0 +1,24 @@
+{
+    "formatVersion": "1",
+    "header": {
+      "description": "Example DOMA for ABAP file formats",
+      "originalLanguage": "en"
+    },
+    "negative_values": false,
+    "case_sensitive": false,
+    "data_type": "CHAR",
+    "length": 3,
+    "decimals": 2,
+    "output_length": 3,
+    "conversion_routine": "",
+    "value_table": "",
```

Comment:
Just a minor. Since these fields are empty, the usually won't be serialized.
```suggestion
```

Reply:
Fixed!